WalletWasabi/Models/UpdateStatus.cs

```diff
@@ -21,9 +23,9 @@ public UpdateStatus(bool backendCompatible, bool clientUpToDate)
 
 		public bool Equals(UpdateStatus other) => this == other;
 
-		public override int GetHashCode() => ClientUpToDate.GetHashCode() ^ BackendCompatible.GetHashCode();
+		public override int GetHashCode() => (13 * ClientUpToDate.GetHashCode()) ^ (47 * BackendCompatible.GetHashCode()) ^ LegalDocumentsVersion.GetHashCode();
```

Comment:
Just a question what are these numbers `13` & `47`?

Reply:
primes

Reply:
You know that we must use numbers. Why 13 and 47 I don't know, probably it's been decided by some smart matematicians that it results in the least collisions or something like that.

Reply:
Thanks for the explanation.